Modules/xASL_module_ASL.m

```diff
@@ -92,7 +92,9 @@
 end
 
 % Make sure to split the M0 from the ASL time-series if needed
-xASL_io_SplitASL_M0(x.P.Path_ASL4D, x.M0PositionInASL4D, x.DummyScanPositionInASL4D);
+if ~isempty(x.M0PositionInASL4D) || ~isempty(x.DummyScanPositionInASL4D)
```

Comment:
Is this part close to settings the default for these parameters (should also be the beginning of `xASL_module_ASL`)? Just to avoid bulky code

Reply:
Defaults are defined above. This was only checking if it makes sense to run the function or not. But this is now moved inside this wrapper. If no indices are provided, it will not continue running the SplitASL function.